evap/evaluation/models.py

```diff
@@ -984,12 +984,20 @@ def textanswer_review_state(self):
     def ratinganswer_counters(self):
         return RatingAnswerCounter.objects.filter(contribution__evaluation=self)
 
+    @property
+    def all_participants_are_external(self):
+        for participant in self.participants.all():
+            if not participant.is_external:
+                return False
+        return True
+
     @property
     def grading_process_is_finished(self):
         return (
             not self.wait_for_grade_upload_before_publishing
             or self.course.gets_no_grade_documents
             or self.course.final_grade_documents.exists()
+            or self.all_participants_are_external
```

Comment:
This might look a bit strange in code. Can you link the issue (just by referencing #2332 in the code in a comment)?